_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+The Competition Team are working closely with the Student Robotics trustees so that we can make the most sensible decision.
+
+## The Competition
+
+At the time of writing, we are carefully considering whether the competition can go ahead. We will be announcing our decision by Wednesday 18th March at the very latest. 
```

Comment:
Probably worth stating that as it stands, we're planning to run the event.

Reply:
Resolved in later commit